Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp

```diff
@@ -159,11 +159,10 @@ void URobotVehicleMovementComponent::UpdateMovement(float InDeltaTime)
                 // Moves the robot up or down, depending on impact position
                 float minDistance = *Algo::MinElement(contactsDistance);
                 minDistance -= RayOffsetUp;
-
                 minDistance = FMath::Min(minDistance, FallingSpeed * InDeltaTime);
 
                 FVector heightVariation = {0., 0., -minDistance};
-                PawnOwner->AddActorWorldOffset(heightVariation, true, &hit, ETeleportType::None);
+                PawnOwner->AddActorWorldOffset(heightVariation, false, &hit, ETeleportType::None);

```

Comment:
any reason to make it false?

Reply:
Here I sometimes got collisions between floor and robot where robot was visibly few centimeters above the floor, although `show collision` terminal command did not show any collisions.
At this point, collision checks between contact points and floor has already been computed so not sweeping is possible. But of course, there are possibilities that floor concavities collide with the robot body parts far from contact points then. In current project, that does not happen in normal cases.